doc/sphinx/Pacemaker_Explained/resources.rst

```diff
@@ -38,15 +32,16 @@ that the author is comfortable with.
 Resource Classes
 ################
 
-Pacemaker supports several classes of agents:
+Pacemaker supports several classes, or standards, of resource agents:
 
 * OCF
 * LSB
 * Systemd
 * Upstart (deprecated)
 * Service
 * Fencing
-* Nagios Plugins
+* Nagios
```

Comment:
Should `alert` count? Looking at `agents.h` and that's the only one not mentioned here. It is special but so is `stonith` in different ways

Reply:
No, alert agents are not *resource* agents

Reply:
Definitely makes sense. The constant is `PCMK_RESOURCE_CLASS_ALERT` so wanted to ask and make sure